plugins/ltac/coretactics.mlg

```diff
@@ -259,13 +259,6 @@ TACTIC EXTEND simple_destruct
 | [ "simple" "destruct" quantified_hypothesis(h) ] -> { simple_destruct h }
 END
 
-(** Double induction *)
-
-TACTIC EXTEND double_induction DEPRECATED { Deprecation.make () }
-| [ "double" "induction" quantified_hypothesis(h1) quantified_hypothesis(h2) ] ->
-  { Elim.h_double_induction h1 h2 }
```

Comment:
You're not removing the internal implementation of `double induction`? Is it used elsewhere?

Reply:
Indeed, it needs to be removed. There are potentially even more cleanups to perform after that.

Reply:
My oversight.  Let me know if I missed some other things.  It would be so helpful to have a tool that would report mli declarations that are not used in the source tree.  (This would require a way to annotate APIs that are only used by external plugins to avoid false positives.)

I didn't update the doc_gram files pending #13770.  I think running `doc_grammar` is not strictly necessary before merging this. Removing the one line from `orderedGrammar` will happen automatically the next time `doc_grammar` is run.  But `rewriting.rst` is in a transitional state so it would get some changes we don't want to apply yet.  I expect @Zimmi48 and I will have that resolved in a week or so.